modules/sno-du-conn-enabling-kdump.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-connected.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-conn-enabling-kdump_{context}"]
+= Enabling kdump
+
+Apply this MachineConfig CR to your cluster to enable the kdump feature that captures critical information in case of a kernel crash.
+
+.Procedure
+
+*  Enable kdump using the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 06-kdump-enable-master
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    systemd:
+      units:
+      - enabled: true
+        name: kdump.service
+  kernelArguments:
+    - crashkernel=256M
```

Comment:
This is now changed to 512M. Can be done in separate doc PR if desired.
https://github.com/openshift-kni/cnf-features-deploy/blob/release-4.10/ztp/source-crs/extra-manifest/06-kdump-master.yaml#L18